desktop/windows/release-notes/index.md

```diff
@@ -23,18 +23,40 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
-## Docker Desktop 4.4.4
-2022-01-24
+## Docker Desktop 4.5.0
+2022-02-10
 
 > Download Docker Desktop
 >
 > [For
 > Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
 > .button .primary-btn }
 
-### Security
+### New
+
+- Add Docker Desktop version to 'docker version' output.
+
+### Upgrades
+
+- [Amazon ECR Credential Helper v0.6.0](https://github.com/awslabs/amazon-ecr-credential-helper/releases/tag/v0.6.0)
 
-- Fixed [CVE-2022-23774](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-23774) where arbitrary files could be moved on the system.
+### Bug fixes and minor changes
+
+- Show correct version number on the title bar when uninstalling.
+- Increased filesystem watch ("inotify") limits by setting `fs.inotify.max_user_watches=1048576` and 'fs.inotify.max_user_instances=8192' in Linux.
```

Comment:
This one is fixing the «too many open files» error as reported in https://github.com/docker/for-mac/issues/6071